docs/web/docs/guides/tutorials/first_task.md

```diff
@@ -236,45 +236,54 @@ Complete this task, and you can review it in the same way as before.
 Now that you've gotten a task running, this section gives a quick overview on some of the components in the configs and `static_test_script.py` that led to the observed behaviors. The goal is to ensure you can write your own run files by the end of the tutorial sections.
 
 ### 3.1 Config registration
-Mephisto wires up to configuration using standard Hydra syntax, but with both `yaml` files (for ease of writing) _and_ structured configs (for ease of documentation). Here's the config we've set up for this example, re-ordered for understanding:
+Mephisto wires up to configuration using standard Hydra syntax, but with both `yaml` files (for ease of writing) _and_ structured configs (for ease of documentation). 
+Here's the config we've set up for this example:
 ```python
 # static_test_script.py
 from mephisto.abstractions.blueprints.static_html_task.static_html_blueprint import (
-    BLUEPRINT_TYPE,
+    BLUEPRINT_TYPE_STATIC_HTML,
+)
+from mephisto.tools.scripts import process_config_and_get_operator, task_script
```

Comment:
I don't think we need `process_config_and_get_operator` in the imports for this example